clients/unshaded/src/main/java/tachyon/client/ClientContext.java

```diff
@@ -96,7 +96,7 @@ public static synchronized TachyonConf getConf() {
   }
 
   /**
-   * @return the ClientMetrics for this client
+   * @return the {@link ClientMetrics} for this client
```

Comment:
Relative path if the import is already present, otherwise full to avoid adding new imports. This is also recommended by checkstyle: http://checkstyle.sourceforge.net/config_imports.html#UnusedImports